lib/property.js

```diff
@@ -14,14 +14,17 @@ var utils = require('./utils');
  * @param {String} property
  * @param {String} value
  * @param {Selector} selector the property originates from
+ * @param {Integer} priority 0 for normal properties, 2 for !important properties.
+ * @param {Array} additional array of integers representing more detailed priorities (sorting)
  * @api public
  */
 
-function Property(prop, value, selector, priority) {
+function Property(prop, value, selector, priority, additionalPriority) {
   this.prop = prop;
   this.value = value;
   this.selector = selector;
   this.priority = priority || 0;
+  this.additionalPriority = additionalPriority;
```

Comment:
As for performances I guess it is better to use "undefined" value and have the "if (this.additionalPriority) {" check when you try to use it. Otherwise I can put the "| []" and then remove the if and let "array.push" to try to push elements even when the array is empty.

BTW I'll do that if this better reflect the current coding style. Commit coming..